src/gui/internallinkwidget.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2022 by Claudio Cambra <claudio.cambra@nextcloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#include "ui_internallinkwidget.h"
+#include "internallinkwidget.h"
+#include "accountstate.h"
+#include "folderman.h"
+#include "theme.h"
+
+#include "QProgressIndicator.h"
+#include <QClipboard>
+#include <QToolButton>
+
+namespace OCC {
+
+Q_LOGGING_CATEGORY(lcInternalLink, "nextcloud.gui.internalink", QtInfoMsg)
```

Comment:
internalink => internallink (like in the class name)

Reply:
Nice catch, fixed